core/domain/stats_jobs_continuous.py

```diff
@@ -411,27 +412,72 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        # pylint: disable=line-too-long
```

Comment:
Here and elsewhere -- don't you need to re-enable after the long line?

Alternatively, you could put the pragma to the right of the long line -- that may be better. There's no problem in making a line that's already long a bit longer :)